### PR TITLE
Restore internal swaps

### PR DESCRIPTION
**Pull request by brunobar79**
Fixes RNBW-####
Figma link (if any):

## What changed (plus any additional context for devs)
Slack thread: https://rainbowhaus.slack.com/archives/CGUDDJYNL/p1666570461390639

## Final checklist

- [ ] Assigned individual reviewers?
- [ ] Added labels? (`critical path`, `release`, `dev QA`)
- [ ] Did you test both iOS and Android?
- [ ] If your changes are visual, did you check both the light and dark themes?
- [ ] Added e2e tests? If not, please specify why
- [ ] If you added new critical path files, did you update the CODEOWNERS file?
- [ ] If no `dev QA` label, did you add the PR to the QA Queue?

### CODE REVIEW

**Review by skylarbarrera — APPROVED** [review]
David Copperfield

File: src/hooks/useSwapDerivedOutputs.ts [hunk 1]
```diff
@@ -111,6 +111,7 @@ const getInputAmount = async (
 
     const rand = Math.floor(Math.random() * 100);
     Logger.debug('Getting quote ', rand, { quoteParams });
+    // @ts-ignore About to get quote
```
Comment (skylarbarrera) [inline src/hooks/useSwapDerivedOutputs.ts:114]: can we add a lil comment that says `do not deleeeet` on these

**Review by jkadamczyk — APPROVED** [review]

File: src/hooks/useSwapDerivedOutputs.ts (hunk 1 above)
Comment (brunobar79) [reply src/hooks/useSwapDerivedOutputs.ts:114]: no

File: src/hooks/useSwapDerivedOutputs.ts [hunk 2]
```diff
@@ -111,6 +111,7 @@ const getInputAmount = async (
 
     const rand = Math.floor(Math.random() * 100);
     Logger.debug('Getting quote ', rand, { quoteParams });
```
Comment (brunobar79) [inline src/hooks/useSwapDerivedOutputs.ts:113]:
```suggestion
    Logger.debug('Getting quote ', rand, { quoteParams });
    // Do not deleeeet the comment below 😤
```

File: src/hooks/useSwapDerivedOutputs.ts [hunk 3]
```diff
@@ -211,7 +212,7 @@ const getOutputAmount = async (
...
```
Comment (brunobar79) [inline src/hooks/useSwapDerivedOutputs.ts:214]:
```suggestion
    Logger.debug('Getting quote ', rand, { quoteParams });
    // Do not deleeeet the comment below 😤
```

**Pull request merged**